.changelog/12057.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+serf: Complete the leave process if boradcasting leave timeout.
```

Comment:
This is in Serf right? It may make sense to note the upgrade here so users can refer to its changelog as well.